shop3/shop3.asd

```diff
@@ -266,12 +266,14 @@ shop3."
     :version (:read-file-form "shop-version.lisp-expr")
     :components ((:module "shop-test-helper"
                           :pathname "tests/"
-                          :components ((:file "common")))
+                          :components ((:file "common")
+                                       (:file "warns-check")))
```

Comment:
I think we should check to see if this macro is already defined by FiveAM (there's a [pull request](https://github.com/lispci/fiveam/pull/81) for it).  So at the top of warns-check, add:

``` lisp
(eval-when (:compile-toplevel :load-toplevel :execute)
    (when (fboundp (uiop:intern* '#:warns :fiveam)) (pushnew :fiveam-warns features)))

#+(not fiveam-warns)
<warns definition>
```
If I knew that this was going to be added at a particular FiveAM version, I could do this in a cleaner way, but 🤷‍♂️

Reply:
I added the check before the warns definition here: https://github.com/shop-planner/shop3/pull/139/commits/ccb211dbd3116aa8d2e85f5fa79ab325f36423e1

Let me know if that's not what you are expecting and I can fix it.

Reply:
The CI tests are failing after adding the check for existence of warns. At a glance, the test failure looks unrelated to the change, so I’m confused at the moment.

I’ll run the tests locally later today to see if I can replicate and debug the test failure.